autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -4,18 +4,89 @@
+    section_number = ''
+    section_term_code = course['term']
+    
+    section_min_credits = course['creditHourLow']
+    section_max_credits = course['creditHourHigh']
 
-def parse_meeting(meeting):
-    " parse_section() must be done first, and pass the data for each meeting here"
+    section_max_enrollment = course['maximumEnrollment']
+    section_current_enrollment = course['enrollment']
+    section_instructor = ''
+    for i in course['faculty']:
+        instructor_name = i['displayName']
+        section_instructor.join(instructor_name)
 
-def parse_instructor(instructor):
-    " parse_course() must be done first, and pass the data for the instructor here "
+    #creates and saves section object
+    s = Section(id=section_id, subject=section_subject, course_num=course_number, section_num=section_number, term_code=section_term_code, min_credits=section_min_credits, max_credits=section_max_credits, max_enrollment=section_max_enrollment, current_enrollment=section_current_enrollment, instructor=section_instructor)
+    s.save()
+
+    #calls parse meeting for each meeting time in the section
+    for i in course['meetingsFaculty']:
+        parse_meeting(i)
+
+def parse_meeting(i):
+    #gets values to create object
+    meeting_id = ''
+    meeting_crn = i['meetingTime']['courseReferenceNumber']
+
+    meeting_building = i['meetingTime']['building']
+    #checks which days classes are on and adds them to attend_days
+    meeting_class_days = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday']
+    attend_days = []
+    for j in meeting_class_days:
+        if i['meetingTime'][j] == True:
+            attend_days.append(j)
+    meeting_start_time = i['meetingTime']['beginTime']
+    meeting_end_time = i['meetingTime']['endTime']
+
+    meeting_class_type = i['meetingTime']['meetingType']
+    meeting_section = ''
+
+    #creates and saves meeting object
+    m = Meeting(id=meeting_id, crn=meeting_crn, building=meeting_building, meeting_days=attend_days, start_time=meeting_start_time, end_time=meeting_end_time, meeting_type=meeting_class_type, section=meeting_section)
+    m.save()
+
+def parse_instructor(course):
+    #gets values to create object
+    for i in course['faculty']:
+        instructor_id = '' 
+        instructor_email = i['emailAddress']
+        instructor_name = i['displayName']
+        #creates and saves instructor object
+        i = Instructor(id=instructor_id, email_address=instructor_email, name=instructor_name)
+        i.save()
 
 def parse_course(course):
-    " Parse a course and save all of its relevant information to the database "
+    course_dept = course['subject']
+    course_number = course['courseNumber']
+    course_title = course['courseTitle']
+    course_term_code = course['term']
+    course_id = generate_course_id(course_dept, course_number, course_term_code)#primary key
+
+    course_desc = ''#long description-could not find in data
+
+    course_prereqs = ''
```

Comment:
Could not find prereqs or coreqs in data. I left them as '' for now so that I could match the model. Honestly, I don't even really think these are necessary though. Aside from checking coreqs to make sure a user registers all required corequistes and giving them a warning if they don't, I don't see any reason why we would need this data. It also seems like a feature that should be implemented later and not prioritized right now.

Reply:
Just made a comment about this but didn't realize you originally said this. You're correct in that we have no current plans for using this data, so it can be removed.